### PR TITLE
Slightly better fatal dialog in CoqIDE.

### PR DESCRIPTION
**Pull request by ppedrot**
This is now of a reasonable size that fits on my screen and it allows copy-pasting the various error information in a reasonable way.

(This was motivated by the recent change in findlib that vomits a dozen of lines of errors at CoqIDE startup.)

### CODE REVIEW

**Comment by ppedrot** [comment]
Visual comparison. I didn't cut the first picture, the borders of the dialog are really offscreen.

Before
shot1

After
shot2

File: ide/coqide/coq.ml [hunk 1]
```diff
@@ -57,43 +57,39 @@ let rec read_all_lines in_chan =
...
+      "Answer", (String.concat "\n" lines);
+      "Exception", Printexc.to_string exn
+    ]
 
 let display_coqtop_answer cmd lines =
-  final_info_popup (List.length lines < 30)
-    ("Coqtop exited\n"^
-     "Command was: "^cmd^"\n"^
-     "Answer was: "^(String.concat "\n  " lines))
+  coq_error_popup ~code:0 ~msg:"Coqtop exited"
```
Comment (Alizter) [inline ide/coqide/coq.ml:88]:
```suggestion
  coq_error_popup ~code:0 ~msg:"coqtop has exited."
```

**Review by Alizter — APPROVED** [review]

File: ide/coqide/coq.ml (hunk 1 above)
Comment (ppedrot) [reply ide/coqide/coq.ml]: I've added a bit of surrounding text to prevent starting a sentence with a non-capitalized word.

**Comment by Alizter** [comment]
OK merging soon

**Comment by Alizter** [comment]
@coqbot merge now

**Comment by herbelin** [comment]
I just discovered the effect of this PR. Very nice.

**Pull request merged**